tests/cypress/e2e/features/shortcuts.js

```diff
@@ -0,0 +1,244 @@
+// Copyright (C) 2024 CVAT.ai Corporation
+//
+// SPDX-License-Identifier: MIT
+
+/// <reference types="cypress" />
+
+context('Customizable Shortcuts', () => {
+    const searchShortcuts = [
+        {
+            title: 'Save the job',
+            description: 'Send all changes of annotations to the server',
+        },
+        {
+            title: 'Switch automatic bordering',
+            description: 'Switch automatic bordering for polygons and polylines during drawing/editing',
+        },
+    ];
+    const taskName = 'A task with markdown';
+    const serverFiles = ['images/image_1.jpg'];
+    const createRectangleShape2Points = {
+        points: 'By 2 Points',
+        type: 'Shape',
+        labelName: 'label 1',
+        firstX: 250,
+        firstY: 350,
+        secondX: 350,
+        secondY: 450,
+    };
+    let taskID = null;
+    let jobID = null;
+
+    const task = {
+        labels: [
+            {
+                name: 'label 1',
+                attributes: [
+                    {
+                        name: 'attribute 1',
+                        mutable: true,
+                        input_type: 'checkbox',
+                        default_value: 'false',
+                        values: ['false'],
+                    },
+                ],
+                type: 'any',
+            }, {
+                name: 'label 2',
+                attributes: [],
+                type: 'any',
+            }, {
+                name: 'label 3',
+                attributes: [],
+                type: 'any',
+            },
+        ],
+        name: taskName,
+        project_id: null,
+        source_storage: { location: 'local' },
+        target_storage: { location: 'local' },
+    };
+    const storage = {
+        server_files: serverFiles,
+        image_quality: 70,
+        use_zip_chunks: true,
+        use_cache: true,
+        sorting_method: 'lexicographical',
+    };
+
+    before(() => {
+        cy.visit('auth/login');
+        cy.login();
+        cy.get('.cvat-tasks-page').should('exist').and('be.visible');
+        cy.url().should('contain', '/tasks');
+        cy.headlessCreateTask(task, storage).then((response) => {
+            taskID = response.taskID;
+            [jobID] = response.jobIDs;
+        });
+    });
+
+    function testSearchShortcuts(searchDescription) {
```

Comment:
Why true/false? It would be more readable if we specify something more concrete (e.g. parameter `where: 'title' | 'description')